### PR TITLE
TOOL!

### PR DESCRIPTION
**Pull request by Very-Soft**

### CODE REVIEW

File: code/game/objects/micro_event.dm [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+			ourmob.resize(size_limit, FALSE, TRUE, TRUE)
+	else
+		if(ourmob.size_multiplier < size_limit)
+			ourmob.resize(size_limit, FALSE, TRUE, TRUE)
+
+	return ..()
+
+/obj/structure/portal_event/resize/preset_shrink_twentyfive
+	shrinking = TRUE
+	size_limit = 0.25
```
Comment (Cameron653) [inline code/game/objects/micro_event.dm:39]: me after I walk into the 25% shrink portal that leads directly in front of a pred:
https://www.youtube.com/watch?v=TvgsGBzuurU

**Review by Cameron653 — APPROVED** [review]
Looks solid.

File: code/game/objects/micro_event.dm (hunk 1 above)
Comment (Very-Soft) [reply code/game/objects/micro_event.dm:39]: U R A LIDDOL GOB!

Comment (Heroman3003) [reply code/game/objects/micro_event.dm:39]: smol

**Pull request merged**